deposit/src/test/java/edu/unc/lib/deposit/normalize/BioMedToN3BagJobTest.java

```diff
@@ -110,18 +109,13 @@ public void test() {
 		Resource primaryResource = (Resource) depositBag.iterator().next();
 		assertNotNull("Main object from the deposit not found", primaryResource);
 
-		assertTrue("Primary resource was not assigned content models to be an aggregate",
```

Comment:
Its good that isAggregate was removed, but we should still verify that the primaryResource is a Work.  That just boils down to a primaryResource.hasProperty call I believe